acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/account/Account.java

```diff
@@ -78,7 +78,7 @@ public static Account create(final EthTransactions eth, final String name) {
     return new Account(eth, name, SIGNATURE_ALGORITHM.get().generateKeyPair());
   }
 
-  static Account fromPrivateKey(
+  public static Account fromPrivateKey(
```

Comment:
Not related to my change, but it makes it easier to use this DSL outside of Besu repo